actions/debootstrap_action.go

```diff
@@ -25,7 +28,9 @@ Optional properties:
 Example:
  components: [ main, contrib ]
 
-- keyring-package -- keyring for packages validation. Currently ignored.
+- keyring-package -- keyring for package validation.
+
+- keyring-file -- keyring file for repository validation.
```

Comment:
Should probably  document that this is relative to the recipe dir if a non-absolute path is given

Reply:
I'll do a follow-up commit. See also #85.